proxy-node-gateway/src/main/java/uk/gov/ida/notification/resources/EidasAuthnRequestResource.java

```diff
@@ -62,7 +63,7 @@ public View handleRedirectBinding(
     @Path(Urls.GatewayUrls.GATEWAY_EIDAS_AUTHN_REQUEST_POST_PATH)
     @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
     public View handlePostBinding(
-            @FormParam(SamlFormMessageType.SAML_REQUEST) String encodedEidasAuthnRequest,
+            @FormParam(SamlFormMessageType.SAML_REQUEST) @ValidBase64SamlAuthnRequest String encodedEidasAuthnRequest,
```

Comment:
Please see above comment about saml parsing.

Reply:
Removed and replaced with a simple Base64 check.